client/components/image-asset/example.js

```diff
@@ -0,0 +1,30 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import DocsExample from 'devdocs/docs-example';
+import ImageAsset from './';
+
+export default class ImageAssetExample extends Component {
+	static defaultProps = {
+		displayName: 'ImageAsset',
+		exampleCode: (
+			<div>
+				<ImageAsset src="/empty-content.svg" alt="" width={ 200 } height={ 200 } />
+			</div>
+		),
+	};
+
+	renderExample() {
+		return this.props.exampleCode;
+	}
+
+	render = () => {
+		return <DocsExample { ...this.props }>{ this.renderExample() }</DocsExample>;
```

Comment:
This `DocsExample` wrapper should be added in the root `DevDocs` component, IMO - then devs could just copy the examples.